core/native-bridge.ts

```diff
@@ -480,6 +480,28 @@ const initBridge = (w: any): void => {
             return win.CapacitorWebFetch(resource, options);
           }
 
+          if (
+            !options?.method ||
+            options.method.toLocaleUpperCase() === 'GET' ||
+            options.method.toLocaleUpperCase() === 'HEAD' ||
+            options.method.toLocaleUpperCase() === 'OPTIONS' ||
+            options.method.toLocaleUpperCase() === 'TRACE'
+          ) {
+            const url = new URL(resource.toString());
```

Comment:
Doesn't it need to check if the url is relative as in the XHR part?

Could be the url patching code be moved to a separate function that does the relative url check and string replacement in a single place so is less error prone?

We could even export it so users could create "proxy" urls for img tag or other html elements.